src/utils.js

```diff
@@ -72,8 +74,16 @@ const version = () => {
 	console.log( `Wikipedia Preview - version ${APP_VERSION} (${GIT_HASH})` )
 }
 
+const getPreviewHtml = ( title, lang ) => {
```

Comment:
Having this inside `utils` is a bit out of place. Utils are low-level. no-dependencies little helpers. They are used by higher level code like api and preview. Now this creates a circular dependency. 

I would recommend creating this function directly in index or in a new file if you can find a good name for it.

Reply:
Makes sense, let's go with directly in index for now

Reply:
> if you can find a good name for it.

yeah, i think the good name here is very important, as this expose the method to everyone with `wikipediaPreview.getPreviewHtml()`, at least we need write some comment/doc to tell why this is here.

Reply:
It's for internal use only. We can comment on the function itself that it is used by the plugin so we remember not to break it but no public documentation.